src/dune_pkg/fetch.ml

```diff
@@ -56,16 +56,59 @@ module Curl = struct
            ([ Pp.textf "curl returned an invalid error code %d" exit_code ] @ stderr)))
     else (
       Path.unlink_no_err stderr;
-      match Int.of_string http_code with
+      (* Since we had to quote the return http_code, we need to strip it *)
+      match
+        let open Option.O in
+        String.drop_prefix_and_suffix ~prefix:"\"" ~suffix:"\"" http_code
```

Comment:
Here's an alternative, possibly more readable (except apparently to the Github syntax highlighting)

```ocaml
let prefix = {|"|} in
let suffix = prefix in
String.drop_prefix_and_suffix ~prefix ~suffix http_code
```

Reply:
I noticed that it's easier just to get the substring of the output.